src/L2GovernorMetadata.sol

```diff
@@ -9,6 +9,7 @@ abstract contract L2GovernorMetadata {
   struct Proposal {
     uint256 voteStart;
     uint256 voteEnd;
+    bool cancelled;
```

Comment:
nit: for bools i prefer `isCancelled`

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/61/commits/a5a7ccdbfe2c0c1b6101e2f4eb8b97ab4d312d1b